kuma/wiki/models.py

```diff
@@ -541,8 +541,7 @@ def schedule_rendering(self, cache_control=None, base_url=None):
         Document.objects.filter(pk=self.pk).update(render_scheduled_at=now)
         self.render_scheduled_at = now
 
-        if (waffle.switch_is_active('wiki_force_immediate_rendering') or
-                not self.defer_rendering):
+        if (not self.defer_rendering):
```

Comment:
``wiki_force_immediate_rendering`` is not active in production. This change turns it on permanently. Only the ``else:`` branch should remain after this change.  Which may be difficult - I'm not sure why ``from . import tasks`` was a deferred import.

Reply:
I was wondering about that when I made the change so I decided to err on the side of caution.